bitnami/mongodb/templates/replicaset/scripts-configmap.yaml

```diff
@@ -137,4 +152,102 @@ data:
     export MONGODB_EXTRA_DATABASES_FILE=""
     export MONGODB_EXTRA_PASSWORDS_FILE=""
     exec /opt/bitnami/scripts/mongodb/entrypoint.sh /opt/bitnami/scripts/mongodb/run.sh
+  {{- if .Values.replicaSetConfigurationSettings.enabled }}
+  replicaSetConfigurationSettings.sh: |-
+    #!/bin/bash
+    # This script to be called when pod starts.
+    # This script sets rs settings which can not be applied via conf file
+
+    function logger ()
```

Comment:
This is my comment

Reply:
I see your comment and thank you for it.

Reply:
Sorry for the long delay.